code/edgeidentity/build.gradle

```diff
@@ -2,11 +2,26 @@ apply plugin: 'com.android.library'
 apply plugin: 'jacoco'
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
+apply plugin: "com.diffplug.spotless"
 
 jacoco {
     toolVersion = rootProject.ext.jacocoVersion
 }
 
+preBuild.dependsOn spotlessApply
```

Comment:
This will apply the formatting when the project is built. I feel this could be convenient, but as there is the pre-commit hook, if this ends up adding too much time to every build we can remove it.